src/composables/useAuthenticator.ts

```diff
@@ -0,0 +1,35 @@
+import { getCurrentInstance, computed, ComputedRef } from 'vue';
```

Comment:
We're adding the composables folder according to the [Vue documentation](https://vuejs.org/guide/reusability/composables.html#what-is-a-composable). If we need to get the BP list somewhere else in the OBE, we'll be able to do so easily.